import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/StorageCommitmentService.java

```diff
@@ -149,8 +149,10 @@ public void run() {
             attrsItem.getString(Tag.ReferencedSOPClassUID));
         queryAttributes
             .setString(Tag.SOPInstanceUID, VR.UI, cmtItem.getInstanceUid());
-        queryAttributes
-            .setString(Tag.SOPClassUID, VR.UI, cmtItem.getClassUid());
+        // current (v1beta1) healthcare api wrongly returns
+        // "SOPClassUID is not a supported instance or study or series level attribute"
+        //queryAttributes
```

Comment:
I'd prefer deleting the code for now, and we can use commit history to add it back once the issue is resolved on the server side.

Reply:
Ok